src/fluentd/deploy/fluentd-config.yaml.template

```diff
@@ -143,4 +144,27 @@ data:
           overflow_action block
         </buffer>
       </store>
+{% endif %}
+{% if (cluster_cfg['internal-storage']['enable']) and (cluster_cfg['postgresql']['enable']) %}
```

Comment:
seems you can use if else...

Reply:
Better use something like:
job-attempt-storage-engine: "elasticsearch"  # set it to "db" to use database backend


---
In reply to: [391421409](https://github.com/microsoft/pai/pull/4164#discussion_r391421409) [](ancestors = 391421409)

Reply:
use cluster_cfg['cluster']['common']['job-attempt-storage-engine']